README.md

```diff
@@ -38,3 +38,18 @@ highlighting, anonymity, and secure authentication.
 - Preferably use `ROCKET_ENV=prod` and set a secret key in `Rocket.toml`
   - See [Rocket docs](https://rocket.rs/guide/configuration/)
 - Reverse proxy and handle `/static/` with a webserver and not the included route
+
+### Usage (docker)
+
+- Clone the repo (`--recursive` for submodules)
+- `echo 'DATABASE_URL=postgres://paste:paste@db/paste' > .env`
+- Copy config.toml to paste.toml and edit `paste.toml`
+- `docker-compose up -d`
+- `docker exec -ti paste_paste_1 sh /root/post-install.sh`
+- `docker exec -ti paste_paste_1 sh /root/start.sh`
+
+To connect to postgres from within docker, run:
+
+```sh
+docker exec -ti paste_paste_1 psql
```

Comment:
nonfunctional as of now: somehow .pgpass seems to be ignored entirely despite the documentation saying all is good.

Reply:
you need to pass the database you want to connect to, `psql paste` would probably work